st2reactor/st2reactor/garbage_collector/base.py

```diff
@@ -196,7 +196,8 @@ def _purge_action_executions(self):
         timestamp_str = isotime.format(dt=timestamp)
         LOG.info('Deleting action executions older than: %s' % (timestamp_str))
 
-        assert timestamp < utc_now
+        if not timestamp < utc_now:
```

Comment:
Weird comparison here.

```suggestion
        if timestamp > utc_now:
```